FatPkg/EnhancedFatDxe/Fat.h

```diff
@@ -84,6 +84,19 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define FAT_FATCACHE_GROUP_MIN_COUNT      1
 #define FAT_FATCACHE_GROUP_MAX_COUNT      16
 
+// For cache block bits, use CPU native size
+#define DIRTY_BLOCKS          UINTN

```

Comment:
It may be cleaner to use a fixed size type such as UINT32 so the logic is the same for all CPU Archs and debugging will be the same when viewing the bitmasks.

Reply:
I agree, I will update this to a fixed type size.